### ISSUE TITLE
General Issues with Trope Website Upgrade to SquareSpace

### ISSUE THREAD

**Issue by akallish** [Tombodil/Trope22#2]
Hello @Tombodil I finally connected the repro to ZenHub so I thought this issue can be for general issues. You can create specific tasks issues and we can have threaded discussions there.

**Comment by akallish** [comment]
@Tombodil hello. Wanted to check in to see how the migration is going.

**Comment by Tombodil** [comment]
@akallish Hey Adam,
Left a Mssg. In short, all the tech heavy lifting is ready to go. I'm updating the kan ban today as I've gotten page integration to work in a vertical slice. Custom html - Jquery coding and css. The integration took more setup and specificity then Squarespace lets on but we're good to go. 

On the building end, I'm literally building a lot of the site blocks out in html as necessary. Over the next week more pages will be updated and integrated as I complete at least one full run of each page type for testing. Once feedback is complete I can fill in same level pages.

**Comment by akallish** [comment]
@Tombodil Thanks for the update and sorry I was unable to answer your call. It all makes sense. 

I did get the invitation to log into SquareSpace which I did. The interface looked confusing so I just chalked it up to this is the sandbox and is a work in progress. 

This weekend I plan to concept the home page and will post it in this repository for comment by you.

**Comment by Tombodil** [comment]
Hey wanted to give a general heads up here. Today off, so been working on the site, will have evenings for the next few days.

Got the fonts to put into squarespace. I need to wire them up through the style editor. About to work on that. 

Built the page types local and am porting over now. Some adjustments are required but all coding and structure works as it's standard html editing. I have to hand it to squarespace. There's a learning curve but it really does handle custom formatting. 

Biggest issue are relative links breaking because Squarespace has their own folder structure. Thankfully it's well documented to it's not a block, just some tedious checking when I drop in a page archtype.

I will be building the main page up over the next days off I have on the 31st - 1st. This is the time in which the page types are being formatted so they are almost ready when I start on the landing page. 

In other words I'll have another check in update by Next Monday, 30th. My estimation is that each page type will be about ready to go so I can build up the homepage.

**Comment by akallish** [comment]
> I will be building the main page up over the next days off I have on the 31st - 1st. This is the time in which the page types are being formatted so they are almost ready when I start on the landing page.

He @Tombodil Thanks for the updates, they are appreciated. I will be posting a PDF of the home page this week. Can you set up a review meeting of your progress? Thanks
